### PR TITLE
add events

### PR DESCRIPTION
**Pull request by plavacquery**
mix events from several datasource,
add category (outing_type) for events

### CODE REVIEW

File: idunn/api/places_list.py [hunk 1]
```diff
@@ -103,7 +112,15 @@ def valid_category(cls, v):
 
 
 class EventQueryParam(CommonQueryParam):
-    pass
+    outing_types: str = None
```
Comment (amatissart) [inline idunn/api/places_list.py]: why plural ? I think a single value is expected here.

Also, this parameter name is too specific in my opinion. On reflection, using `type` may be confusing, because `"type":"event"` is already present in the response. I would stick to `category`, similarly to the /places endpoint.

File: idunn/api/places_list.py [hunk 2]
```diff
@@ -177,10 +194,20 @@ def get_events_bbox(bbox, query_params: http.QueryParams):
             detail={"message": e.errors()}
         )
 
+    print(params.outing_types)
+    current_outing_lang = params.outing_types
+
+    if params.outing_types:
+        lang = params.lang.split(',')[0]
+        current_outing_lang = params.outing_types.get(lang)
```
[the remark this answers is not in the document]
Comment (plavacquery) [reply idunn/api/places_list.py]: no it use for the translation in outing_types (category)

File: tests/test_environment_weather.py [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+        )
+
+        res = Weather.from_es(
+            Admin(testee),
+            lang='en'
+        )
+
+    assert res == Weather(**{
+        'temperature': 291.89,
+        'weatherPic': '01d',
```
[the remark this answers is not in the document]
Comment (plavacquery) [reply tests/test_environment_weather.py]: It is the icon code value, do not find any better value

File: idunn/blocks/events.py [hunk 4]
```diff
@@ -53,6 +53,7 @@ class DescriptionEvent(BaseBlock):
     description = validators.String(allow_null=True)
     free_text = validators.String(allow_null=True)
     price = validators.String(allow_null=True)
+    tags = validators.String(allow_null=True)
```
[the remark this answers is not in the document]
Comment (plavacquery) [reply idunn/blocks/events.py]: the tags are an entire string,redo it as a list

File: idunn/api/places_list.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (plavacquery) [reply idunn/api/places_list.py]: indeed it use for es query because, it find out the word concert, show, exhibition, .. in the appropriate language

File: idunn/api/places_list.py (hunk 1 above)
Comment (plavacquery) [reply idunn/api/places_list.py]: ok I will redefine it as 'outing_category'

File: idunn/api/kuzzle.py [hunk 5]
```diff
@@ -93,19 +93,38 @@ def fetch_event_places(self, bbox, size) -> list:
...
                             }
                         }
-                    },
-                    'must': {
-                        'range': {
-                            'date_end': {
-                                'gte': 'now/d',
-                                'lte': 'now+31d/d'
-                            }
-                        }
```
[the remark this answers is not in the document]
Comment (plavacquery) [reply idunn/api/kuzzle.py]: no, I will remove lte for date_end and add a lte now+7d for date start

[the remark this answers is not in the document]
Comment (plavacquery) [reply idunn/api/kuzzle.py]: in fact yes I will add a better filter on date

Do not take into account my previous remark

File: idunn/api/utils.py [hunk 6]
```diff
@@ -42,6 +42,7 @@
         GradesBlock
     ],
     LIST: [
+        AirQuality,
```
Comment (amatissart) [inline idunn/api/utils.py]: why this change ?

File: idunn/utils/default_settings.yaml [hunk 7]
```diff
@@ -75,6 +75,11 @@ BLOCK_AIR_QUALITY_ENABLED: True
 KUZZLE_CLUSTER_URL:
 KUZZLE_REQUEST_TIMEOUT: "0.5" # seconds
 
+# weather API
+WEATHER_API_URL:
+WEATHER_API_KEY:
+WEATHER_REQUEST_TIMEOUT: "0.5" #seconds
+
```
Comment (amatissart) [inline idunn/utils/default_settings.yaml]: this should not be part of this PR

File: tests/fixtures/api_weather_response.json [hunk 8]
```diff
@@ -0,0 +1,15 @@
...
+    "visibility": 10000, 
+    "wind": {"speed": 2.6, "deg": 230}, 
+    "clouds": {"all": 0}, 
+    "dt": 1568044180, 
+    "sys": {"type": 1, "id": 6540, "message": 0.0078, "country": "FR", "sunrise": 1568006291, "sunset": 1568053101},
+    "timezone": 7200, 
+    "id": 2978048,
+    "name": "Saint-Merri",
+    "cod": 200
+}
```
Comment (amatissart) [inline tests/fixtures/api_weather_response.json]: this should not be part of this PR

File: tests/test_events_informations.py [hunk 9]
```diff
@@ -72,7 +74,8 @@ def test_event_information_missing_fields():
         type="event_description",
         description="15h-16h [LECTURES D'ALBUMS] Pour les petits (3-6 ans). Accès libre et gratuit.",
         free_text=None,
-        price="Gratuit"
+        price="Gratuit",
+        tags=['']
```
Comment (amatissart) [inline tests/test_events_informations.py]: I think, we would expect an empty list instead

File: idunn/api/utils.py (hunk 6 above)
Comment (plavacquery) [reply idunn/api/utils.py]: my mistake

File: idunn/utils/default_settings.yaml (hunk 7 above)
Comment (plavacquery) [reply idunn/utils/default_settings.yaml]: indeed

File: tests/fixtures/api_weather_response.json (hunk 8 above)
Comment (plavacquery) [reply tests/fixtures/api_weather_response.json]: Indeed

File: idunn/api/places_list.py [hunk 10]
```diff
@@ -103,7 +112,15 @@ def valid_category(cls, v):
 
 
 class EventQueryParam(CommonQueryParam):
-    pass
+    outing_category: str = None
```
Comment (amatissart) [inline idunn/api/places_list.py]: after discussion, to rename to `category`
